### PR TITLE
CCITCARBON-217 teflo init examples

### PR DESCRIPTION
**Pull request by shay6**

### CODE REVIEW

**Comment by shay6** [comment]
I copied ready-made samples from this repo to the teflo_init folder.

File: teflo_init/execute/README.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
+Shell
```
Comment (rujutashinde) [inline teflo_init/execute/README.rst]: What does shell mean here? Should the heading be Execute_Example ?

File: teflo_init/orchestrate/README.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
+Basic
```
Comment (rujutashinde) [inline teflo_init/orchestrate/README.rst]: What does  Basic mean here , should the heading be Orchestrate_Example ?

File: teflo_init/provision/README.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
+Static
```
Comment (rujutashinde) [inline teflo_init/provision/README.rst]: What does  Static  mean here , should the heading be Provision_Example ?

**Review by rujutashinde — COMMENTED** [review]
I think rest looks ok, just a couple of comments

File: teflo_init/execute/README.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+=====
+
+This example demonstrates how you can execute tests against test machines
+using teflo.
```
Comment (JunqiZhang0) [inline teflo_init/execute/README.rst:5]: I think this is not **example demonstrates how you can execute tests against test machines**, but teflo_init folder?

File: teflo_init/execute/README.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+This example below has teflo installed in a virtual environment named teflo.
+Please replace this virtual environment with the virtual environment you
+created for teflo if it differs.
+
+Pre-Run
+-------
+
+First lets review the content of the `scenario descriptor file <scenario.yml>`_.
+This file contains a provision section which contains one host resource
```
Comment (JunqiZhang0) [inline teflo_init/execute/README.rst:17]: Why do we need all these lines for teflo init folder?

File: provision/static/scenario.yml [hunk 6]
```diff
@@ -5,7 +5,7 @@ description: define a static resource to be used throughout teflo
 
 provision:
   - name: static-node
-    role: node
+    groups: node
```
Comment (JunqiZhang0) [inline provision/static/scenario.yml]: You need to fix this in a separate PR, as this is not related to the teflo_init

File: teflo_init/execute/teflo.cfg [hunk 7]
```diff
@@ -0,0 +1,4 @@
+[defaults]
+log_level=info
+workspace=.
+data_folder=.teflo
```
Comment (JunqiZhang0) [inline teflo_init/execute/teflo.cfg:4]: This should be more detailed, you should include how users would use **credentials, timeout, concurrency settings and etc**

**Review by JunqiZhang0 — CHANGES_REQUESTED** [review]

File: teflo_init/execute/teflo.cfg (hunk 7 above)
Comment (rujutashinde) [reply teflo_init/execute/teflo.cfg:4]: @JunqiZhang0 for this example I do not think ther eis a need to add the details, because there are other examples if the user has to look for how to set up the credentials. Here we are just giving them basic workspace with a running local example which does not need credentials as such. Maybe we can always add a note to the reAdme , "for more examples checke out the examples repo"

File: provision/static/scenario.yml (hunk 6 above)
Comment (rujutashinde) [reply provision/static/scenario.yml]: @JunqiZhang0 , this example within the teflo_init directory, so i think we dont need to have a separate PR for this. We can put a new PR , for correcting the original example, if thats what you are talking about. The changes here are only within the teflo_init directory

File: teflo_init/execute/README.rst (hunk 1 above)
Comment (shay6) [reply teflo_init/execute/README.rst]: fixed

File: teflo_init/orchestrate/README.rst (hunk 2 above)
Comment (shay6) [reply teflo_init/orchestrate/README.rst]: fixed

File: teflo_init/provision/README.rst (hunk 3 above)
Comment (shay6) [reply teflo_init/provision/README.rst]: fixed

File: provision/static/scenario.yml (hunk 6 above)
Comment (shay6) [reply provision/static/scenario.yml]: I will correct the original files in a separate PR.
this file is from the original files, so it will be handle in the new PR.

File: teflo_init/execute/teflo.cfg (hunk 7 above)
Comment (shay6) [reply teflo_init/execute/teflo.cfg:4]: I added a note at the end of the README files "for more examples check out the examples repo" with a link to the examples repo.

File: provision/static/scenario.yml (hunk 6 above)
Comment (shay6) [reply provision/static/scenario.yml]: This is the new PR  to correct the original files
https://github.com/RedHatQE/teflo_examples/pull/5

**Review by JunqiZhang0 — APPROVED** [review]
LGTM

**Review by rujutashinde — APPROVED** [review]
